src/styles/index.scss

```
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 100%;
+  position: relative;
+}
+
+.stopwatch__center {
+  width: 5vmin;
+  height: 5vmin;
+  border-radius: 100%;
+  background-color: #f6a603;
+  position: absolute;
+  z-index: 10;
+}
```

You can use scss nesting everywhere
```suggestion
.stopwatch {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 80vmin;
  height: 80vmin;
  border: 1vmin dotted #000;
  border-radius: 100%;
  position: relative;
  
  &__center {
    width: 5vmin;
    height: 5vmin;
    border-radius: 100%;
    background-color: #f6a603;
    position: absolute;
    z-index: 10;
  }
}

```